create_elasticsearch_secrets.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
I suggest either explicitly naming this file `create_elasticsearch_secrets_aws.sh` or similar, or expanding it to the point that we can support more cloud providers in the future and it's not S3-specific.

Reply:
e.g. script can be called as `create_elasticsearch_secrets.sh aws` or `create_elasticsearch_secrets.sh gcp` (with anything other than `aws` for now throws a "not implemented" error)

Reply:
I'd go for now with the explicit file naming since maybe there's other implementations required etc. etc. so rather redo the docs later and unify the scripts than spend now the time for something that may never happen.

Reply:
It's part of the subfolder `aws/dual-region/scripts`, which I think should be clear that's provider specific.
If and when we add more providers, we can potentially abstract logic into shared code functions.